controllers/controller.js

```diff
@@ -10,42 +10,33 @@ mongoose.Promise = require('bluebird');
 mongoose.connect('mongodb://test:test@ds257627.mlab.com:57627/first-blog-mern', { useMongoClient: true });
 
 const dataService = new service();
-module.exports = app => {
-
-//    const homeFunc = (req, res) => {
-//        Post.find({}, function(err, data){
-//            if (err) throw (err);
-//            res.render('posts', {posts: data});
-//        });
-//    };
+module.exports = (app) => {
```

Comment:
This controller should be implemented using [express routes](https://developer.mozilla.org/en-US/docs/Learn/Server-side/Express_Nodejs/routes)